### PR TITLE
Introduce more guidance on getting start contributing

### LINKED ISSUE #495: Blog post on how to get started with contributing to the TAG app delivery - WG Platforms
Idea: Create a blog post about how can people get started on contributing to the TAG app delivery. Largely on the line of creating an issue, sharing the idea, joining the meetings, discussing the outcomes, TG, etc., and eventually publishing the blog post.

### PR DESCRIPTION
**Pull request by abangser**
We have made awesome strides by introducing more contributing guides to the website in #519.

This closes #495 by introducing both a detailed blog post and an accompanying issue template which can support organising people's thoughts and provide the TAG with more information to support new ideas.

This is heavily modelled after the sandbox template but of course needs to be much more light weight. One thing that has been removed but may need to be re-added is a reference to who owns IP from any contribution.

Big thanks to @techmaharaj for a lot of the content and the first review of this PR.

### CODE REVIEW

File: website/content/en/blog/contributing-to-wg-platforms.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title:  Getting started with contributing in WG Platforms
+slug:   contributing-to-wg-platforms
+date:   2023-12-22 12:00:00 +0000
+author: Abby Bangser & Atul Sharma
```
Comment (techmaharaj) [inline website/content/en/blog/contributing-to-wg-platforms.md:5]:
```suggestion
author: Abby Bangser & Atulpriya Sharma
```

File: website/content/en/blog/contributing-to-wg-platforms.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* Vendor or other promotional pitches
+* Topics not related to application delivery or cloud native technologies
+* Discriminatory or abusive content
+
+
+## Step 1 - Submission
+
+We would encourage you to generate a GitHub issue and open a Slack thread with your idea. You can use [this link](https://github.com/cncf/tag-app-delivery/issues/new?template=community-contribution.md) to create an issue and also start a thread on the [CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf) in the [#wg-platforms channel](https://cloud-native.slack.com/archives/C020RHD43BP).
+
+This will present you with a templated issue where you can follow the prompts. You will first need to write a descriptive title, then answer each of the questions in the description field. If you find one is not relevant that is OK, just write that and there is always a chance to chat more on these things after your initial submission.
```
Comment (techmaharaj) [inline website/content/en/blog/contributing-to-wg-platforms.md:50]:
```suggestion
The GitHub issue will present you with a template where you can follow the prompts. First, You must write a descriptive title, then answer each question in the description field. If you find one is not relevant that is OK, write that and there is always a chance to chat more on these things after your initial submission.

For Slack, you can start a new thread in this format: [Proposal]<My_New_Idea>
```

File: website/content/en/blog/contributing-to-wg-platforms.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+During the review process, we’ll do the following checks:
+* _Basic grammar, syntax, and language check_ - we suggest using a tool like Grammarly before submitting for review.
+* _Technical correctness_ - we’ll validate the technical accuracy of what you’ve written.
+* _Vendor Pitches/Promotional links_ - we’ll carefully go through the content to ensure there’s no promotional material.
+
+## Step 4 - Final Approval and Publishing
+
+After the review is complete, and there’s a consensus from everyone that this is good to go, the WG lead will initiate the process of raising a PR and merging it.
+
+What next?
```
Comment (techmaharaj) [inline website/content/en/blog/contributing-to-wg-platforms.md:78]:
```suggestion
## What next?
```

**Review by techmaharaj — COMMENTED** [review]
Looks good to go. Left minor suggestions.

**Review by lianmakesthings — CHANGES_REQUESTED** [review]
Love this! Thanks so much for your work.

One thing: the "Getting started with contributing in WG Platforms" doesn't seem to show up in the deploy preview. Not sure what's going on there. Maybe I'm missing something.

**Comment by RobertKelly** [comment]
Very detailed with encouraging guidance. Looks great to me! :shipit:

**Comment by abangser** [comment]
Doh! @lianmakesthings I made an assumption that it would take some time to get reviews so I post-dated it to Dec 22. This meant it wouldn't show up on the blog. I just updated the date to today and confirmed locally that it shows up now. The preview should also be up soon!

**Comment by lianmakesthings** [comment]
> Doh! @lianmakesthings I made an assumption that it would take some time to get reviews so I post-dated it to Dec 22. This meant it wouldn't show up on the blog. I just updated the date to today and confirmed locally that it shows up now. I expected a preview deploy to occur again but it keeps coming up as canceled 🤔 This may need to be either on trust or verified locally. So sorry!
No worries. Maybe you could just put the PR in draft mode if you're not ready for an "official" review yet. I tend to try to get reviews out of the way asap, so maybe I'm just too quick (or too thorough 😂 )

**Comment by abangser** [comment]
This was definitely ready for review. I just didn't realise that was how the dates worked :til:

So there are two things I guess... 1) is the content reasonable and 2) does it show up on the site. Unfortunately the preview deploy isn't coming up for my change and I am not sure why not. So for #2 it unfortunately can only be validated locally 😢

**Comment by lianmakesthings** [comment]
@joshgav @cjyabraham I remember you were trying to figure out why new commits in a PR don't trigger a rebuild of the deploy preview. Did you come to any solutions?

**Comment by cjyabraham** [comment]
The deploy preview is here: https://deploy-preview-523--tag-app-delivery.netlify.app/blog/contributing-to-wg-platforms/

I'm not sure why netlify is reporting that the deploy is cancelled since it is built and seems up-to-date to me. You can find the link to deploy previews by inspecting a commit and clicking on "Details":
Screenshot 2023-12-21 at 7 51 30 AM

**Review by lianmakesthings — APPROVED** [review]

**Pull request merged**